app/models/user.rb

```diff
@@ -1,16 +1,22 @@
 # frozen_string_literal: true
 require 'common/models/base'
+require 'mvi/messages/find_candidate_message'
+require 'mvi/service'
 
 class User < RedisStore
   NAMESPACE = REDIS_CONFIG['user_store']['namespace']
   REDIS_STORE = Redis::Namespace.new(NAMESPACE, redis: Redis.current)
   DEFAULT_TTL = REDIS_CONFIG['user_store']['each_ttl']
+  MVI_SERVICE = VetsAPI::Application.config.mvi_service
 
   # id.me attributes
   attribute :uuid
   attribute :email
   attribute :first_name
+  attribute :middle_name
   attribute :last_name
+  attribute :gender
+  attribute :dob, Common::UTCTime
```

Comment:
It could be a string, and MVI does end up using it as a String, but then our storage of DOB is defined by MVI and I think we should keep it generic so if future classes need it they can format it as they see fit :)